src/platform/plugins/private/presentation_panel/public/panel_component/presentation_panel_error_internal.tsx

```diff
@@ -120,26 +145,113 @@ export const PresentationPanelErrorInternal = ({ api, error }: PresentationPanel
     );
   }
 
+  const { isLandscape, isNarrow } = useMemo(() => {
+    const { width, height } = panelSize ?? { width: Infinity, height: Infinity };
+    return {
+      isLandscape: width > height && height < euiTheme.breakpoint.s / 3,
+      isNarrow: height < euiTheme.breakpoint.s / 10,
+    };
+  }, [panelSize, euiTheme.breakpoint.s]);
+
   return (
-    <EuiEmptyPrompt
-      body={
-        searchErrorDisplay?.body ?? (
-          <EuiText size="s" css={errorTextStyle}>
-            <Markdown data-test-subj="errorMessageMarkdown" readOnly>
-              {error.message?.length
-                ? error.message
-                : i18n.translate('presentationPanel.emptyErrorMessage', {
-                    defaultMessage: 'Error',
-                  })}
-            </Markdown>
-          </EuiText>
-        )
-      }
-      data-test-subj="embeddableStackError"
-      iconType="warning"
-      iconColor="danger"
-      layout="vertical"
-      actions={actions}
-    />
+    <div
+      ref={panelRef}
+      data-test-subj="embeddableError"
+      css={[styles.fullWidthAndHeight, styles.outerWrapperStyles]}
+    >
+      {isNarrow ? (
+        <NarrowError error={error} />
+      ) : (
+        <EuiFlexGroup direction="column" gutterSize="l" css={styles.innerWrapperStyles}>
```

Comment:
This is the standard embeddable error:

| Landscape | Portrait |
|--------|--------|
| <img width="584" height="197" alt="image" src="https://github.com/user-attachments/assets/b2fb21c1-5d70-47e6-a8ae-5d1957a475ec" /> | <img width="805" height="352" alt="image" src="https://github.com/user-attachments/assets/cd5d408a-9a20-4153-8a2e-4aa9897db886" /> |